x-pack/test/integrations_manager_api_integration/config.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I don't see where this file is imported or called. Is it a convention of the framework or specified elsewhere?

Reply:
This is my best guess at the convention how to setup tests in this directory that need specific configuration passed to the test server. It is used when you run the tests manually as described in the initial PR description. I'm still in the process of finding out how (or: if) they are picked up during CI runs.

Reply:
Update: see https://github.com/elastic/kibana/pull/45781/commits/fd80f57fbe2a2605b47da3cfeeaf3af61f3d3a03 how to integrate the new tests in the CI runs.

Reply:
oh, right. Now I recall that you have to add multiple entry points.

Good luck 🤞